### PR TITLE
Lua command for setting initial TLM values

### LINKED ISSUE #175: API function for setting initial TLM effort variables
For example something like this:
oms2_setTLMInitialEffort("tlmmodel", "submodel:port", <value>)

### PR DESCRIPTION
**Pull request by robbr48**
## Related Issues

Fixes #175 

## Purpose

API function for setting intial value/effort in TLM interfaces

## Approach

Added a Lua function, which inserts variables in a map. When sockets are initialized, initial values are taken from the map if they exist, otherwise from the FMI modelDescription.

## Type of Change

- New feature (non-breaking change which adds functionality)

## Checklist

- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings

---

### CODE REVIEW

File: src/OMSimulatorLua/OMSimulatorLua.c [hunk 1]
```diff
@@ -816,6 +816,48 @@ static int OMSimulatorLua_oms2_setTLMSocketData(lua_State *L)
...
+    double values[6];
+    values[0] = lua_tonumber(L,3);
+    values[1] = lua_tonumber(L,4);
+    values[2] = lua_tonumber(L,5);
+    values[3] = lua_tonumber(L,6);
+    values[4] = lua_tonumber(L,7);
+    values[5] = lua_tonumber(L,8);
+    status = oms2_setTLMInitialValues(cref, subref, values, 6);
+  }
+  lua_pushnumber(L, status);
```
Comment (lochel) [inline src/OMSimulatorLua/OMSimulatorLua.c]: Use `lua_pushinteger` instead; see #194

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/360//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#196

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
**[Linux:](https://test.openmodelica.org/hudson/job/OMSimulator-PR/360/)** The test suite is unstable. :worried:

**Comment by lochel** [comment]
@robbr48 Looks like a memory leak in the TLM implementation, right? I think this error pops up every now and then.

**Comment by robbr48** [comment]
@lochel Yes, but I have not been able to recreate it on my local machine, so it is difficult to debug. Since it always happens after the simulation is finished, it must be related to some destructor code I guess.

**Comment by lochel** [comment]
ASan could possibly pinpoint the problem.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/363//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#196

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :sun_with_face:

**Pull request closed without merging**